src/Daemon.cpp

```diff
@@ -215,6 +217,7 @@ int main(int argc, char *argv[]) {
 
     try {
         Database db(argv[1]);
+        spdlog::info("UrsaDB v{}", get_version());
```

Comment:
What about:
```
spdlog::info("                          _ _");
spdlog::info(" _   _ _ __ ___  __ _  __| | |__");
spdlog::info("| | | | '__/ __|/ _` |/ _` | '_ \");
spdlog::info("| |_| | |  \__ \ (_| | (_| | |_) |");
spdlog::info(" \__,_|_|  |___/\__,_|\__,_|_.__/");
```
:thinking:
(plus \\ escapes) 
(not sure though).

Reply:
Not a bad idea, although I'll leave it out for now.